src/pages/components/sheet.mdx

```diff
@@ -0,0 +1,200 @@
+---
+title: Sheet
+description: >
+  The Sheet component is a container that acts as a supplementary view, providing access to important contextual information.
+package: '@zendeskgarden/react-chrome'
+components:
+  - chrome/src/elements/sheet/Sheet.tsx
+subcomponents:
+  - chrome/src/elements/sheet/components/Body.tsx
+  - chrome/src/elements/sheet/components/Title.tsx
+  - chrome/src/elements/sheet/components/Description.tsx
+  - chrome/src/elements/sheet/components/Header.tsx
+  - chrome/src/elements/sheet/components/Footer.tsx
+  - chrome/src/elements/sheet/components/FooterItem.tsx
+  - chrome/src/elements/sheet/components/Close.tsx
+---
+
+<Usage>
+  <Use>
+    <ul>
+      <li>To display additional information that aids the primary view</li>
+    </ul>
+  </Use>
+  <Misuse>
+    <ul>
+      <li>
+        <Markdown>
+          For an experience where the primary view is obscured to display supplementary information
+          or actions, use a [Drawer](/components/drawer)
+        </Markdown>
+      </li>
+    </ul>
+  </Misuse>
+</Usage>
+
+## How to use it
+
+### Default
+
+A typical usage of a Sheet component.
+
+import SheetDefault from '../../examples/chrome/SheetDefault.tsx';
+import SheetDefaultCode from '!!raw-loader!../../examples/chrome/SheetDefault.tsx';
+
+<CodeExample code={SheetDefaultCode}>
+  <SheetDefault />
+</CodeExample>
+
+### Footers
+
+Sheet footers comes in three variations: no footer, footer with an [Anchor](/components/anchor),
+and footer with [Buttons](/components/button).
+
+import SheetFooters from '../../examples/chrome/SheetFooters.tsx';
+import SheetFootersCode from '!!raw-loader!../../examples/chrome/SheetFooters.tsx';
+
+<CodeExample code={SheetFootersCode}>
+  <SheetFooters />
+</CodeExample>
+
+### Headers
+
+Sheet headers come in three variations: no title, title, and title with description.
+
+import SheetHeaders from '../../examples/chrome/SheetHeaders.tsx';
+import SheetHeadersCode from '!!raw-loader!../../examples/chrome/SheetHeaders.tsx';
+
+<CodeExample code={SheetHeadersCode}>
+  <SheetHeaders />
+</CodeExample>
+
+### Placement
+
+Sheets can be placed at the start or end of the container. End is the default placement.
+
+import SheetPlacement from '../../examples/chrome/SheetPlacement.tsx';
+import SheetPlacementCode from '!!raw-loader!../../examples/chrome/SheetPlacement.tsx';
+
+<CodeExample code={SheetPlacementCode}>
+  <SheetPlacement />
+</CodeExample>
+
+### Size
+
+Sheets default to 380px in width. You can customise the width based on your intended
```

Comment:
@Francois-Esquire Just caught this now. "customise" --> "customize" 
We use American English spelling. It's mentioned in my [How to write for Garden guidelines](https://zendesk.atlassian.net/wiki/spaces/GARDEN/pages/5372510361/How+to+write+for+Garden) which got published after we designed this documentation but going forward I'll be really pushing these guidelines for both designers and engineers so we can all catch them earlier in the process. cc: @james-sann @jon-kerwin 

I was able to catch most of these things when in it was in the documentation phase but sometimes they still slip through...

Reply:
Ahh I missed that as well! Good catch Diana, thank you!